conda_envs/py3.11.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.7
+  - python=3.11
```

Comment:
This is great that you guys are trying to get this to work with python 3.11! However, not all python packages are fully testes with python 3.11 yet. So I'd suggest going down 1 for now to python 3.10.